raft/src/main/java/org/apache/kafka/raft/RecordAppender.java

```diff
@@ -30,7 +30,7 @@
      * This method must be thread-safe.
      *
      * @param records The records to append to the log
-     * @return A future containing the base offset and epoch of the appended records (if successful)
+     * @return A future containing the last offset and epoch of the appended records (if successful)
```

Comment:
This is one of the changes I made for append API.

Reply:
Should we name the parameter as lastOffset in the `OffsetAndEpoch` struct?

Reply:
The `OffsetAndEpoch` is used in many other places where the offset is not the last offset.